servicex_local/science_images.py

```diff
@@ -7,7 +7,6 @@
 
 from .logging_decorator import log_to_file
 
```

Comment:
Please use `black` to do your formatting. That will keep this second line here. You can set this up in vscode so that it will always apply the formatting updates automatically.